### ISSUE TITLE
Consider moving Postbank to its own repository/plugin

### ISSUE THREAD

**Issue by omarkohl** [MirkoDziadzka/ofxstatement-1822direkt#2]
It may be confusing and unexpected for users to find that by installing 1822direkt they now have two different plugins for two different banks. Also if someone is looking for the Postbank plugin they may not find it (e.g. it is not listed in the ofxstatement repository).

**Comment by MirkoDziadzka** [comment]
Thanks for the proposal.

I will probably rename this to `ofxstatement-german-banks` instead and add stuff like my [plugin for the Amazon Credit Card](https://github.com/MirkoDziadzka/ofxstatement-bubbas/blob/master/src/ofxstatement/plugins/lbbamazon.py) and other existing plugins. I think this may make more sense for a user with multiple bank accounts.

On the other hand, this would it make harder to add a 3rd party plugin for one of the banks.

I will think about it.

**Comment by omarkohl** [comment]
I see you already created ofxstatement-germany.

I think that approach has two disadvantages:

* It creates a bottleneck (you) which may become a problem if at some point you become unresponsive or unwilling and someone wants to fix or add a plugin for a certain bank. This could of course be solved by adding people that have merge permissions, but you would have to manage/choose those people.
* You and your repository may be considered trustworthy. If you merge any plugin for any german bank without reviewing the code this may be a security risk (bank statement information should be considered sensitive data). If you do review all the code this creates work for you which brings us back to the bottleneck

**Comment by mirkodziadzka-avi** [comment]
@omarkohl I agree that these 2 disadvantages exist. The advantage is: It saves me time for now.

If you want to fork my ofxstatement repos, I'm fine with this and would point to you as the primary source.

**Comment by MirkoDziadzka** [comment]
@omarkohl I agree that these 2 disadvantages exist. The advantage is: It saves me time for now.

If you want to fork my ofxstatement repos, I'm fine with this and would point to you as the primary source.
